docs/content-server-api/CONTENT_SERVER_API.md

```diff
@@ -5,6 +5,7 @@
 * Audit: `GET /audit/:type/:entityId`
 * Challenge: `GET /challenge`
 * Contents per hashId: `GET /contents/:hashId`
+* [Get active Entity Id per hashId](contents/hashId/active-entity/get.md) : `GET /contents/:hashId/active-entity`
```

Comment:
Should be update it now that we will return many entities?

Reply:
Yes